### ISSUE TITLE
Add support for /shared/authz/users API

### ISSUE THREAD

**Issue by pbardon** [F5Networks/f5-common-python#1352]
The f5-common-python sdk does not currently have support for querying and creating users via the REST endpoint:

/shared/authz/users

I am planning to submit a pull request to add this functionality, but am creating this issue first as suggested in the contribution guide.

**Comment by caphrim007** [comment]
Have you tried /tm/auth/user? That's the one I usually use for such a situation

**Comment by pbardon** [comment]
Yes, and that endpoint does work with the SDK. This new endpoint is the REST Framework endpoint for user management, and this is the endpoint that can be used to reset the admin password during initial setup due to the recent changes to mark the admin password as expired by default.
